mesonbuild/minstall.py

```diff
@@ -463,8 +463,8 @@ def do_symlink(self, target: str, link: str, full_dst_dir: str) -> bool:
         abs_target = target
         if not os.path.isabs(target):
             abs_target = os.path.join(full_dst_dir, target)
-        if not os.path.exists(abs_target):
-            raise RuntimeError(f'Tried to install symlink to missing file {abs_target}')
+        #if not os.path.exists(abs_target):
+        #    raise RuntimeError(f'Tried to install symlink to missing file {abs_target}')
```

Comment:
Huh, why does this need to be commented out? Don't we install destinations before symlinks already?

Reply:
Opened a TODO review comment myself about this, see above.

Reply:
Ah, good point!